benchmark/README.md

```diff
@@ -31,8 +33,145 @@ npm run benchmark -- -u 4 -e 15 -r 5
 - `npm run benchmark` to run the benchmark with the default setting, to see the available flags run
 - `npm run benchmark -- --help`
 
-## Comming soon
+## Custom Tasks
+
+DiscoJS currently provides several pre-define popular tasks such as titanic, simple-face and cifar10. In order
+to understand how to add your own custom task, we will go over how we added simple-face to discojs.
+
+### Task
+
+The `Task` class is the first piece of the puzzle, this contains all the crucial information from training to mode.
+
+The task class for simple face can be found in [discojs/src/tasks/simple_face.ts](../discojs/src/tasks/simple_face.ts),
+and the contents look as follows. (For brevity we have replaced some lines with ...).
+
+Note that the preprocessFunctions are not yet supported via the Task object, we will explain after how to add a it
```

Comment:
instead of not supported, it's just that preprocessing must be done in/after the dataloader

Reply:
yes, but right now `preprocessFunctions` does not do anything, it's just a placeholder, so I wanted to make that clear.